extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -93,7 +104,61 @@ class InstrumentationService {
           }
         });
       }
+    } else if (eventType === AnalyticsEventType.CLICK) {
+      this.ensureClickListener_();
+      this.clickObservable_.add(
+          this.createSelectiveListener_(listener, opt_selector));
+    }
+  }
+
+  ensureClickListener_() {
+    if (!this.clickHandlerRegistered_) {
+      this.clickHandlerRegistered_ = true;
+      this.win_.document.documentElement.addEventListener(
+          'click', this.onClick_.bind(this));
+    }
+  }
+
+  /**
+   * @param {!Event} e
+   */
+  onClick_(e) {
+    this.clickObservable_.fire(e);
+  }
+
+  /**
+   * @param {!Function} listener
+   * @param {string=} opt_selector
+   */
+  createSelectiveListener_(listener, opt_selector) {
+    const wrappedListener = e => {
+      if (opt_selector) {
```

Comment:
I'm a bit concerned about sending clicks for all elements when selector is empty. I think it's easy enough to specify '*' if the publisher really interested in this.